### ISSUE TITLE
DropArea shows "undefined" while editing a ModelForm without input_file inputed

### ISSUE THREAD

**Issue by yanhuixie** [mbraak/django-file-form#891]
![image](https://github.com/mbraak/django-file-form/assets/1049615/a2fef769-ff8b-45b5-b737-95ee6f167abf)
Yes, I have just change input_file of Example Model to "blank=True, null=True". 
```python
class Example(models.Model):
    fs = FileSystemStorage(location=settings.MEDIA_ROOT)

    title = models.CharField(max_length=255)
    input_file = models.FileField(max_length=255, upload_to="example", storage=fs, blank=True, null=True)

    def __str__(self):
        return self.title
```

Because we make it not required that the file to be uploaded.

```javascript
    const getInitialFiles = fieldName => {
      const data = getInputValue(getUploadsFieldName(fieldName, getPrefix()));
      if (!data) {
        return [];
      }
      return JSON.parse(data);
    };
```
[      if (!data) {](https://github.com/mbraak/django-file-form/blob/34572dd23c4a3a0274a2c8d422adc1b7808e6ad6/django_file_form/static/file_form/file_form.js#L6234) I think It (!data) could be the cause of the problem. In this scenario, the "data" is [{}], that an empty object within an empty array.

**Comment by mbraak** [comment]
I'll have a look

**Comment by mbraak** [comment]
I added a pr with a fix: #892. I plan to merge the pr soon.

**Comment by mbraak** [comment]
This is fixed in version 3.7.0
